### PR TITLE
Add new /drafts/disruptions endpoint into swagger

### PR DESCRIPTION
**Pull request by tchevily**
# Description

This PR adds new `/drafts/disruptions` endpoint documentation into swagger

Issue link: BOT-3342

This is new feature

## How to test

Here are the following steps to test this pull request:

Use swagger editor at https://editor.swagger.io/

You should also check in SLATE mode https://canaltp.github.io/Chaos/

### CODE REVIEW

**Comment by dvdn** [comment]
- Semantic error at paths./drafts/disruptions.post.requestBody.$ref
requestBody $refs cannot point to '#/components/schemas/…', they must point to '#/components/requestBodies/…'
--> line 1691

- Structural error at paths./drafts/disruptions/{id}.get.parameters.2
should have either a `schema` or `content` property
--> line 1732

- Structural error at paths./drafts/disruptions/{id}.put.parameters.2
should have either a `schema` or `content` property
--> line 1753

- Semantic error at paths./drafts/disruptions/{id}.put.requestBody.$ref
requestBody $refs cannot point to '#/components/schemas/…', they must point to '#/components/requestBodies/…'
--> line 1758

- Structural error at paths./drafts/disruptions/{id}.delete.parameters.2
should have either a `schema` or `content` property
--> line 1800

File: documentation/swagger.yml [hunk 1]
```diff
@@ -2524,6 +2700,120 @@ components:
...
+          type: string
+          format: uuid
+          description: Disruption draft ID
+        reference:
+          type: string
+          maxLength: 250
+          description: Publication reference label
+        author:
+          $ref: "#/components/schemas/author"
+        cause:
```
Comment (Netmisa) [inline documentation/swagger.yml:2716]: Maybe we can use `causeId` like `severityId` here https://github.com/CanalTP/Chaos/pull/498/files#diff-56097d5bc49b9e1b81da3a6e83cfa07218e5df6bb189ad96cd5ed4cdb6dba375R2742

File: documentation/swagger.yml [hunk 2]
```diff
@@ -2524,6 +2700,120 @@ components:
...
+        timeZone:
+          type: string
+          example: Europe/Paris
+    disruption_draft_impact_channel:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        label:
```
Comment (Netmisa) [inline documentation/swagger.yml:2811]: I think we don't need this.
We use only `Id` channel to match the correct message.

File: documentation/swagger.yml (hunk 1 above)
Comment (Netmisa) [inline documentation/swagger.yml:2716]:
```suggestion
        causeId:
```

File: documentation/swagger.yml [hunk 3]
```diff
@@ -2524,6 +2700,120 @@ components:
...
+        reference:
+          type: string
+          maxLength: 250
+          description: Publication reference label
+        author:
+          $ref: "#/components/schemas/author"
+        cause:
+          type: string
+          format: uuid
+          maxLength: 250
```
Comment (Netmisa) [inline documentation/swagger.yml:2719]:
```suggestion
```

File: documentation/swagger.yml (hunk 2 above)
Comment (tchevily) [reply documentation/swagger.yml:2810]: we need it, since form contains label as well

**Review by Msaglier — APPROVED** [review]

File: documentation/swagger.yml [hunk 4]
```diff
@@ -2524,6 +2700,119 @@ components:
...
+          type: string
+          example: Europe/Paris
+    disruption_draft_impact_channel:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        label:
+          type: string
```
Comment (Netmisa) [inline documentation/swagger.yml:2811]:
```suggestion
```

**Review by Netmisa — APPROVED** [review]

**Review by dvdn — APPROVED** [review]

**Pull request merged**